src/tendencies/hyperdiffusion.jl

```diff
@@ -104,11 +113,14 @@ function hyperdiffusion_tendency!(Yₜ, Y, p, t)
         end
     end
 
-    @. Yₜ.c.uₕ -= κ₄ * divergence_damping_factor * C12(wgradₕ(divₕ(ᶜ∇²uₕ)))
-    # Without the C12(), the right-hand side would be a C1 or C2 in 2D space.
+    @. Yₜ.c.uₕ -= κ₄ * divergence_damping_factor * C12(wgradₕ(divₕ(ᶜ∇²u)))
+    # Without the C123(), the right-hand side would be a C1 or C2 in 2D space.
     if point_type <: Geometry.Abstract3DPoint
-        # TODO: Are these vector conversions correct when there is topography?
-        @. Yₜ.c.uₕ += κ₄ * C12(wcurlₕ(C3(curlₕ(ᶜ∇²uₕ))))
+        ᶜχ₁₂₃ = @. C123(curlₕ(C12(ᶜ∇²u))) + C123(curlₕ(C3(ᶜ∇²u)))
+        @. Yₜ.c.uₕ += C12(wcurlₕ(C12(ᶜχ₁₂₃))) + C12(wcurlₕ(C3(ᶜχ₁₂₃)))
+        # Reuse the buffer variable here so we don't need an extra one
+        @. ᶜ∇²uᵥ = C3(wcurlₕ(C12(ᶜχ₁₂₃))) + C3(wcurlₕ(C3(ᶜχ₁₂₃)))
+        @. Yₜ.f.u₃ += ᶠinterp(ᶜ∇²uᵥ)
```

Comment:
`ᶜ∇²uᵥ` is a fourth-order diffusion tendency applied to `Yₜ.f.u₃`; naming might be confusing in the future; I suppose it's a tradeoff between re-using buffer vars or assigning more variables to cache.

Reply:
Maybe we need another name then.

Reply:
Perhaps we should replace the 2 with a 4